frontend/src/libs/components/modal-window/modal-window.tsx

```diff
@@ -0,0 +1,20 @@
+import { Portal } from 'react-portal';
+
+import closePopUpIcon from '~/assets/img/close-pop-up-icon.svg';
+
+import styles from './styles.module.scss';
+
+const ModalWindow: React.FC = () => {
+  return (
+    <Portal>
+      <div className={styles.backdrop} />
+      <div className={styles.modal}>
+        <button className={styles.modalPopUpClose}>
+          <img alt="close" src={closePopUpIcon} />
+        </button>
```

Comment:
We already have `IconButton` component - enhance Icons map with `close` svg icon and use `IconButton`